tools/taskcluster/flank.yml

```diff
@@ -34,15 +33,13 @@ gcloud:
     - /sdcard/screenshots
   performance-metrics: true
 
-  # test-targets will be pulled dynamically from package.  DO NOT MODIFY
-  test-targets:
-TEST_TARGETS
-
   device:
-    - model: sailfish 
-      version: 26 
+    - model: blueline
+      version: 28
 
 flank:
+  project: moz-fx-mobile-firebase-testlab
+
   # test shards - the amount of groups to split the test suite into
   # set to -1 to use one shard per test.
   testShards: -1 
```

Comment:
~20 tests or so, feel free to nudge this to 4 shards (5 tests per shard) here while the file is touched

Reply:
👍 Updated it to `4`.